### PR TITLE
updated shared decision making text in the intake note

### PR DESCRIPTION
**Pull request by glilly**
New language added to the Intake Note

### CODE REVIEW

File: routines/SAMINOT1.m [hunk 1]
```diff
@@ -388,20 +388,31 @@ d OUT("")
...
+ d OUT("current diagnosis or obvious symptoms suggestive of lung cancer, has been educated ")
+ d OUT("today about the estimated risk for lung cancer, the possibility of a cure or life ")
+ d OUT("prolonging if an early lung cancer were to be found during screening, the possibility of ")
+ d OUT("imaging abnormalities not being lung cancer, the possibility of complications from ")
+ d OUT("additional diagnostic procedures, and the approximate amount of radiation exposure ")
+ d OUT("associated with each screening procedure. In addition, the Veteran has been educated ")
+ d OUT("today about the importance of avoiding exposure to cigarette smoke, available tobacco ")
+ d OUT("cessation programs and available lung screening services at this VA facility. Education ")
+ d OUT("material was provided to the veteran.")
+ ;d OUT(" ")
```
Comment (djdenv) [inline routines/SAMINOT1.m]: Why not just delete all these comments? Git maintains the history of the changes so you don't have to keep this cluttering the code base.

Comment (glilly) [reply routines/SAMINOT1.m]: I predict that Phoenix will want their own language in their CPRS notes, to be consistent with what they've been doing for years. So I left their language in the code, ready to be conditionally delivered.

Comment (dlwicksell) [reply routines/SAMINOT1.m]: One possible solution that might be cleaner, is to make all of the text a double colon comment (;; ...), having the different sets of text that might be wanted, stored under separate entry tags, and use $TEXT to grab and write out the correct text, with a flag in a configuration global allowing sites to use whatever text they want of the options.

**Comment by glilly** [comment]
Good idea David. If it turns out that we need to customize the intake notes
per site, your suggestion would be a good way to do it.

gpl

**Comment by dlwicksell** [comment]
George, were you planning on making the change I suggested, in this pull request, or as a later enhancement?

File: routines/SAMINOT1.m (hunk 1 above)
Comment (dlwicksell) [inline routines/SAMINOT1.m]: Are you planning to move to a ;; $TEXT solution for this code as well?

Comment (glilly) [reply routines/SAMINOT1.m]: I don't plan to make that change unless Phoenix requests a site specific intake note. Then I would make the change.

**Pull request merged**